_glossary/conditional/user-is-logged-in-start.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: side-nav
+title: User Is logged In Start
+last-updated: 01-03-2013
+nav-position: 17
+parent: conditional
+tag: <!--WDK:isloggedin:start-->
```

Comment:
This is actually already a tag...aaha.

`<!--WDK:if:plugin:useraccount:isloggedin-->`